src/components/Table/body/enumerableCell.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledCell from './styled/cell';
+import StyledCellContent from './styled/cellContent';
+import StyledEnumerable from './styled/enumerable';
+
+export default function EnumerableCell({ rowNumberOffset, rowIndex }) {
+    const rowNumber = rowNumberOffset + rowIndex;
+    return (
+        <StyledCell scope="row" tabIndex={-1}>
+            <StyledCellContent>
+                <StyledEnumerable title={rowNumber}>{rowNumber}</StyledEnumerable>
+            </StyledCellContent>
+        </StyledCell>
+    );
+}
```

Comment:
take a look here:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Lists_and_Counters/Using_CSS_counters
https://css-tricks.com/almanac/properties/c/counter-increment/
I think we should use this since it will be more simple and much more performance

Reply:
applying....